clients/csrf/csrf.go

```diff
@@ -0,0 +1,50 @@
+package csrf
+
+import (
+	"github.com/cloudfoundry-incubator/multiapps-cli-plugin/clients/csrf/csrf_parameters"
+	"net/http"
+	"sync"
+	"time"
+)
+
+const CookieHeader = "Cookie"
+
+type Csrf struct {
```

Comment:
csrf is not very descriptive, maybe rename to
CsrfTokenHelper
CsrfTokenUpdater